app/controllers/assets_controller.rb

```diff
@@ -254,6 +254,7 @@ def print_assets
   end
 
   def show_plate
```

Comment:
Means we get a 404 if someone accesses a non-plate at show plate, rather than a 500.